package.json

```diff
@@ -32,30 +32,30 @@
     "@babel/plugin-transform-runtime": "^7.4.4",
```

Comment:
Are the `package.json` changes wanted?

Reply:
Yes since i have merged https://github.com/kuzzleio/kuzzle/pull/1356 into this pr